tests/test_cliCmdPruneDatasets.py

```diff
@@ -349,7 +349,7 @@ def test_purgeWithDisassociate(self):
             exPruneDatasetsExitCode=1
         )
 
-    @patch.object(lsst.daf.butler.Registry, "getCollectionType",
+    @patch.object(lsst.daf.butler.registry.SqlRegistry, "getCollectionType",
```

Comment:
I guess this would also need patching other registry classes if we ever want to use this test with non-SqlRegistry?

Reply:
Yes, but for now I think it's fine because this is a command line tool test and not really a registry test.